### DISCUSSION TITLE
How to enable file icons by default

### DISCUSSION THREAD

**Discussion by msaquib17** [eclipse-theia/theia#11890] [Q&A] [answered]
On opening theia app on browser we need to enable icons from File->Preferences->File Icon Theme. Is there a way we can enable files by default from the code?

**Comment by msujew** [comment, marked as answer]
Hey @msaquib17,

you can set the default file icon theme by setting the `defaultIconTheme` property in the `package.json` of your application:

https://github.com/eclipse-theia/theia/blob/90982d2edbe1e7aac776501897f92ca4c873c576/examples/browser/package.json#L6-L15

**Reply by msaquib17** [reply]
@msujew can you please give an example in which block and what should be written?

**Reply by msujew** [reply]
It should look something like this:

```json
"theia": { 
   "frontend": { 
     "config": { 
       "applicationName": "Theia Browser Example", 
       "defaultIconTheme": "theia-file-icons"
     } 
   } 
 }, 
```

`theia-file-icons` is the default file icon pack that is distributed with Theia. You can change this to the ID of any contributed file icon theme (i.e. from vscode extensions).

**Reply by msaquib17** [reply]
Thanks
